cli/src/markdownlint-cli2/default.markdownlint-cli2.cjs

```diff
@@ -0,0 +1,29 @@
+// @ts-check
+const _ = require('lodash');
+
+('use strict');
+
+const { env } = require('process');
+var custom_config = null;
+if (env.CUSTOM_CONFIG) {
+  custom_config = require(env.PWD + '/' + env.CUSTOM_CONFIG);
+}
+
+const default_config = {
+  config: {
+    MD007: {
+      indent: 4,
+    },
+    MD013: false,
+    MD033: {
+      allowed_elements: ['Tabs', 'TabItem', 'CodeBlock'],
+    },
```

Comment:
True that. This was introduced my because of the research specs mostly to incentivise using proper markdown features if possible. But yeah guess I need to remove it for now. Will think about it

Reply:
Or like you said, implement our own rule based on this one, or open a PR to let us specifically disallow tags too. If I have spare time I'll look at the latter.

Reply:
Ok, will remove it for now then 👍

Reply:
```suggestion
```